css/index.css

```diff
@@ -143,4 +212,59 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+/* About Section */
+
+.cta-about {
+    margin: 20px;
+    padding: 20px;
+    border-bottom: 1px dashed black;
+}
+
+.info-section {
+    display: flex;
+    justify-content: space-between;
+}
+
+.info-section img {
+    padding: 15px;
+}
+
+.info-section .top-card .card {
+    display: flex;
+    flex-direction: column;
+    padding: 15px;
+    border-bottom: 1px dashed gray;
+    width: 415px;
+    margin: 15px 0;
+}
+
+.info-section .bottom-card .card {
+    display: flex;
+    flex-direction: column;
+    padding: 15px;
+    border-bottom: 1px dashed black;
+    width: 415px;
+    margin: 15px 0;
```

Comment:
Great use of Flexbox and styling to maintain form amongst the cards. Think about how you can possibly capture and style all four cards with one instance of CSS styling.